internal/subshell/cmd/cmd.go

```diff
@@ -149,7 +149,7 @@ func (v *SubShell) Activate(prj *project.Project, cfg sscommon.Configurable, out
 	if prj != nil {
 		env := sscommon.EscapeEnv(v.env)
 		var err error
-		if v.rcFile, err = sscommon.SetupProjectRcFile(prj, "config.bat", ".bat", env, out, cfg); err != nil {
+		if v.rcFile, err = sscommon.SetupProjectRcFile(prj, "config.bat", ".bat", env, out, cfg, false); err != nil {
```

Comment:
Note: my initial thought was to pass in the subshell object (`v`), but this introduces an import cycle in `sscommon`, so I just pass a boolean.